liquibase-service/src/main/resources/db/changelog/db.rdb_modern.changelog-16.1.yaml

```diff
@@ -533,4 +533,28 @@ databaseChangeLog:
       changes:
         - sqlFile:
             path: 026-sp_std_hiv_datamart_postprocessing-001.sql
+            splitStatements: false
+  - changeSet:
```

Comment:
Not for this review but more towards a full-fledged discussion on Liquibase and how scripts are ordered- These views would be helpful to have towards the start of the dimension post-processing stored procedures so they can be referenced.

Reply:
@upasanapattnaik-eq when the scripts are run it should follow this order irrespeticve of the release
1. tables
2. Views or Routines
3. Stored procedures

Reply:
That is the reason why we maintain the database object level folders @upasanapattnaik-eq

Reply:
Unfortunately, we add that all into one single changelog for rdb_modern and it is chronological to when we add new scripts. So would this would be the 84th object to execute.

Reply:
@sveselev @upasanapattnaik-eq  Should we have changelog by db?

Reply:
@alog-enquizit or @sveselev might have some better insight on the deployment steps for Liquibase and if that is possible. Looping in @alog-enquizit